docs/10_utilities/fill-queue/fill-queue-how-to.md

```diff
@@ -0,0 +1,100 @@
+---
+content_title: How-To Setup And Use Fill-Queue
+link_text: How To Setup and Use Fill-Queue
+---
+
+## Summary
+
+This how-to demonstrates the set up and use of [Fill-Queue application](index) running locally.
+
+## Prerequisites
+
+Before you begin, make sure you have the following information or have completed the following referenced procedures:
+
+1. `nodeos` runs locally.
+2. `nodeos` has the state history plugin enabled. \
+For instructions on how to start `nodeos` with history plugin enabled consult
+[the following documentation](https://developers.eos.io/manuals/eos/v2.1/nodeos/plugins/state_history_plugin/index/).
+3. RabbitMQ runs locally.
```

Comment:
there is no requirement to run rabbitmq locally. you can connect to it via network